### ISSUE TITLE
Reloading jobs after Recurring-Integrations-Scheduler service is restarted

### ISSUE THREAD

**Issue by TBVAdacta** [microsoft/Recurring-Integrations-Scheduler#95]
### Expected behavior and actual behavior.
When RIS service is restarted to reload all previously configured jobs.

### Steps to reproduce the problem.
Open RIS and connect to localhost 
Create an import project
Add project to schedule
Wait for project to be executed
Close RIS user interface
Open RIS user interface and connect to localhost 
Previously configuret import job appears in the grid
Close RIS user interface
Restart RIS service
Open RIS and connect to localhost 
No previously configured import jobs appear in the grid.

### Specifications like the version of RIS and platform and application version of D365FO.
RIS version: 2.7.1
Number of jobs: 1
D365FO version: Microsoft Dynamics 365 for Finance and Operations (10.0.6) (10.0.234.10001)
Installed platform version : Update30 (7.0.5407.35455)

### More informations
In the Program files folder where Recurring-Integrations-Scheduler is installed, the Schedule file is still showing the job that was configured, but it seams that this file is not reloaded after the RIS service is restarted.

**Comment by TomekMelissa** [comment]
Does RIS app prompts you to save the schedule file when you are closing it? Are you sure RIS service is installed in the same folder as the app and Schedule file?

**Comment by TBVAdacta** [comment]
1. Are you sure RIS service is installed in the same folder as the app and Schedule file? 
Yes.
Application location: "C:\Program Files (x86)\Recurring Integrations Scheduler\RecurringIntegrationsScheduler.exe"
Service location: "C:\Program Files (x86)\Recurring Integrations Scheduler\Quartz.Server.exe"

2. Does RIS app prompts you to save the schedule file when you are closing it? Yes. we saved the Schedule file into another folder (not the same as where RIS is installed). 
Now we saved it into the same folder and it works.
